### ISSUE TITLE
Is still alive?

### ISSUE THREAD

**Issue by juancaceres** [priaid-eHealth/symptomchecker#13]
Is this software still alive? Somebody have information about this company?

**Comment by fabiants** [comment]
Yes, we are still active. You can subscribe to our service at www.ApiMedic.com
